rtsf-at-checkout-device-scale/Dockerfile

```
@@ -19,18 +19,18 @@ RUN go mod download
 COPY . .
 
 RUN go mod tidy
-RUN go build -tags no_zmq -o device-scale cmd/main.go
+RUN CGO_ENABLED=0 go build -tags no_zmq -o device-scale cmd/main.go
 
 # Next image - Copy built Go binary into new workspace
-FROM alpine:3.16
+FROM alpine:3.17
 
 LABEL license='SPDX-License-Identifier: BSD-3-Clause' \
-    copyright='Copyright (c) 2018 - 2022: Intel'
+    copyright='Copyright (c) 2018 - 2023: Intel'
 
 WORKDIR /
 COPY --from=builder /device-scale/device-scale /device-scale
-COPY --from=builder /device-scale/cmd/res/configuration.toml /res/configuration.toml
+COPY --from=builder /device-scale/cmd/res/configuration.yaml /res/configuration.yaml
 COPY --from=builder /device-scale/cmd/res/profiles/cas-scale.yaml /res/profiles/cas-scale.yaml
-COPY --from=builder /device-scale/cmd/res/devices/device-list.toml /res/devices/device-list.toml
+COPY --from=builder /device-scale/cmd/res/devices/device-list.yaml /res/devices/device-list.yaml
 
-CMD [ "/device-scale","--cp=consul://edgex-core-consul:8500","--registry","--confdir=/res"]
+CMD [ "/device-scale","--cp=consul://edgex-core-consul:8500"]
```

Still can have `--registry`
```suggestion
CMD [ "/device-scale","--cp=consul://edgex-core-consul:8500","--registry"]
```